ros/processor/insights_engine_result_consumer.py

```diff
@@ -114,7 +126,28 @@ def process_report(self, host, reports):
                     fqdn=host['fqdn'],
                     rule_hit_details=reports,
                     number_of_recommendations=rec_count,
-                    state=SYSTEM_STATES[state_key]
+                    state=SYSTEM_STATES[state_key],
+                    instance_type=performance_record.get('instance_type')
+                )
+                LOG.info(
+                    f"{self.prefix} - System created/updated successfully: {host['id']}"
+                )
+
+                performance_utilization = {
+                    'memory': system_details['mem_utilization'],
+                    'cpu': system_details['cpu_utilization'],
+                    'io': system_details['io_utilization']
+                }
+
+                get_or_create(
+                    db.session, PerformanceProfile, ['system_id', 'report_date'],
+                    system_id=system.id,
+                    performance_record=performance_record,
+                    performance_utilization=performance_utilization,
+                    report_date=datetime.datetime.utcnow().date()
+                )
+                LOG.info(
+                    f"{self.prefix} - Performance profile created successfully: {host['id']}"
```

Comment:
```suggestion
                    f"{self.prefix} - Performance profile created successfully for the system: {host['id']}"
```

Reply:
@saltgen Apart from your commits, can you update [this line](https://github.com/RedHatInsights/ros-backend/pull/138/files#diff-0c17bcf0cdabfe68b39e608007e0195be1c3ae63f6395c4f9bd809e003089e26R164) replacing `host['fqdn']` to `host['id']` or another relevant key to display the inventory id and not fqdn since in all the logs elsewhere we are referring to the inventory id and not the fqdn. So, to keep the logging helpful to debug and consistent, I would suggest replacing the key.

Reply:
Sure, change pushed

Reply:
Sure, change pushed